From 0a10d1c8feee3d5e6cf2b0faa7483f3259ca00f5 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Fri, 14 Sep 2018 19:22:38 +0800
Subject: [PATCH 4657/5242] MLK-19589-02 driver: clk: add imx6ulz clock
 support

commit  76fd604d28f6c8bbb30dd25a8203c11885ab96b4 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add i.MX6ULZ clock driver support. i.MX6ULZ clock
tree is same as i.MX6ULL. so reuse the i.MX6ULL
clock compatible check.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx6ul.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/clk/imx/clk-imx6ul.c b/drivers/clk/imx/clk-imx6ul.c
index f9ac484..2888b84 100644
--- a/drivers/clk/imx/clk-imx6ul.c
+++ b/drivers/clk/imx/clk-imx6ul.c
@@ -123,7 +123,8 @@ static inline int clk_on_imx6ul(void)
 
 static inline int clk_on_imx6ull(void)
 {
-	return of_machine_is_compatible("fsl,imx6ull");
+	return of_machine_is_compatible("fsl,imx6ull") ||
+		of_machine_is_compatible("fsl,imx6ulz");
 }
 
 static void __init imx6ul_clocks_init(struct device_node *ccm_node)
-- 
1.7.9.5

